packages/web/examples/CategorySearch/src/index.js

```diff
@@ -27,6 +27,9 @@ const Main = () => (
 					enableRecentSearches
 					enablePopularSuggestions
 					size={5}
+					showClear
```

Comment:
@mohdashraf010897 You can remove these changes.

Reply:
done